ca_gazebo/launch/create_april_tags.launch

```diff
@@ -1,15 +1,18 @@
 <launch>
 	<arg name="env" default="april_tags"/>
- 	<param name="create1/amcl/initial_pose_x" value="5"/>
-  	<param name="create1/amcl/initial_pose_y" value="5"/>
+ 	<param name="create1/amcl/initial_pose_x" value="3"/>
+  	<param name="create1/amcl/initial_pose_y" value="3"/>
   	<param name="create1/amcl/initial_pose_a" value="1"/>
 
- 	<param name="create2/amcl/initial_pose_x" value="-5"/>
-  	<param name="create2/amcl/initial_pose_y" value="-5"/>
+ 	<param name="create2/amcl/initial_pose_x" value="-3"/>
```

Comment:
Do we need more than one robot?

Reply:
I don't know how the `apriltag_ros` package will work spawning two robots with the same tag. In that case, we should pass the tag as a block parameter.